components/lock.rst

```diff
@@ -150,6 +150,51 @@ This component also provides two useful methods related to expiring locks:
 ``getExpiringDate()`` (which returns ``null`` or a ``\DateTimeImmutable``
 object) and ``isExpired()`` (which returns a boolean).
 
+The Owner of The Lock
+---------------------
+
+Locks that are acquired for the first time are owned[1]_ by the ``Lock`` instance that acquired
```

Comment:
Shouldn't it be :class:`Symfony\\Component\\Lock\\Lock`? just asking

Reply:
I figured this should be clear from the docs but I can add that.

Reply:
> Shouldn't it be :class:Symfony\\Component\\Lock\\Lock? just asking

Yes, **but** before this generated a link to the Symfony API which isn't available anymore and now it only links to the plain file at GitHub.
I would propose to keep it like this for now 👍